src/python/tests_extended/test_export_to_onnx.py

```diff
@@ -365,15 +501,41 @@ def test_export_to_onnx(estimator, class_name):
         (onnx_file_size != 0) and
         (onnx_json_file_size != 0) and
         (not 'cannot save itself as ONNX' in output.stdout)):
-            exported = True
 
-    if exported and SHOW_ONNX_JSON:
-        with open(onnx_json_path) as f:
-            print(json.dumps(json.load(f), indent=4))
+        exported = True
+
+        print('ONNX model path:', onnx_path)
+
+        if SHOW_ONNX_JSON:
+            with open(onnx_json_path) as f:
+                print(json.dumps(json.load(f), indent=4))
+
+        # Verify that the output of the exported onnx graph
+        # produces the same results as the standard estimators.
+        if isinstance(estimator, BasePredictor):
+            result_expected = estimator.predict(dataset)
+        else:
+            result_expected = estimator.transform(dataset)
+
+        if isinstance(result_expected, pd.Series):
+            result_expected = pd.DataFrame(result_expected)
+
+        try:
+            onnxrunner = OnnxRunner(model_file=onnx_path)
+            result_onnx = onnxrunner.fit_transform(dataset)
```

Comment:
>result_onnx [](start = 12, length = 11)

Just a heads up: we will need to add onnx results validation from python ORT run as well.